cores/nRF5/pulse.c

```diff
@@ -27,27 +27,25 @@ extern unsigned long countPulseASM(const volatile uint32_t *port, uint32_t bit,
  * or LOW, the type of pulse to measure.  Works on pulses from 2-3 microseconds
  * to 3 minutes in length, but must be called at least a few dozen microseconds
  * before the start of the pulse. */
-uint32_t pulseIn(uint32_t pin, uint32_t state, uint32_t timeout)
+uint32_t pulseIn(uint32_t ulPin, uint32_t state, uint32_t timeout)
 {
   // cache the port and bit of the pin in order to speed up the
   // pulse width measuring loop and achieve finer resolution.  calling
   // digitalRead() instead yields much coarser resolution.
   // PinDescription p = g_APinDescription[pin];
-  uint32_t bit = 1 << pin; //p.ulPin;
+  uint32_t bit = 1 << digitalPinToPin(ulPin);
```

Comment:
Note for later, this can become `digitalPinToBitMask ` once https://github.com/sandeepmistry/arduino-nRF5/pull/432 (or equivalent) is merged.